packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -273,6 +273,14 @@ function Pending(props) {
           </PendingButtonComponent>
         </PendingButtonContainerComponent>
 
+        <PendingButtonContainerComponent {...props}>
+          <PendingButtonComponent
+            {...props}
+            aria-label="Aria label on button">
```

Comment:
for this particular story on Chrome VO, the "pending" is never getting announced. my guess is because the aria-label on the button is never getting updated, and so the screenreader doesn't announce anything since nothing has changed presumably. i think a similar thing is happening with the "Notifications" pending button where the aria-label is not getting updated so "pending" is never announced.